src/components/TextField/TextField.tsx

```diff
@@ -37,11 +40,26 @@ class TextField extends React.Component<TextFieldProps & DebouncedOnChangePrivat
         suffix={this.props.suffix}
         underlined={this.props.underlined}
         onChanged={this.props.unifiedOnChange}
+        componentRef={this.getRef}
         {...getBaseTextFieldProps(this.props)}
       />
     );
   }
 
+  public focus = () => {
+    const el = this.fabricTextFieldElement;
+    if (el) {
+      const valueLength = this.props.value ? this.props.value.length : 0;
+
+      el.focus();
+      el.setSelectionRange(valueLength, valueLength);
```

Comment:
the browser's default behavior with `element.focus()` is to place the cursor at the user's previous location. is there a reason we're manually moving the cursor to the end instead? not saying this is the wrong choice, just wondering what the reason is and if we should ask for UX guidance here

Reply:
In the case I'm implementing, we're going from a disabled to enabled at the same time, and at the end seemed like the right place. 

I can see not always wanting this behavior though, in which case we may also want to expose something like setSelectionRange, or a simplified version of it. 

I've posted on yammer asking for UX guidance.